networkx/algorithms/planarity.py

```diff
@@ -761,6 +775,12 @@ class PlanarEmbedding(nx.DiGraph):
     For a half-edge (u, v) that is orientated such that u is below v then the
     face that belongs to (u, v) is to the right of this half-edge.
 
+    See Also
+    --------
+    check_planarity :
+        Preferred way to check if an existing graph is planar. Also a more
+        convenient way to create a `PlanarEmbedding` from an existing graph.
+
```

Comment:
Now that `is_planar` is working its way into the library, we might want to advertise that over `check_planarity`.
```suggestion
    is _planar : 
        Preferred way to check if an existing graph is planar.
    check_planarity :
        A convenient way to create a `PlanarEmbedding`. If not planar, it returns a subgraph that shows this.

```